packages/console/src/components/AppContent/components/Sidebar/components/Item/index.tsx

```diff
@@ -1,9 +1,10 @@
 import classNames from 'classnames';
-import React, { ReactChild } from 'react';
+import React, { MouseEventHandler, ReactChild, useState } from 'react';
 import { TFuncKey, useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
 
 import { getPath } from '../../utils';
+import Contact from '../Contact';
```

Comment:
I thought it is not suitable to add the `Contact` to the `Item` component.
Maybe we can extend the `Item` to support `to do sth. on click`, and place this `Contact` in the `AppContent`.
Control the `Contact` just like the way we use the `Toast`, or append it to our page when `contact_us` is clicked?

Reply:
updated, PTAL